runtime/src/bank_forks.rs

```diff
@@ -82,7 +83,10 @@ impl BankForks {
     }
 
     pub fn banks(&self) -> HashMap<Slot, Arc<Bank>> {
```

Comment:
Fine to leave this as a follow-up task, but I don't think we need this interface to be entirely unchanging.
None of the call-sites care that it's specifically an `Arc<Bank>`, and actually most don't even require ownership of the `HashMap` itself!

I checked and this compiles just fine:
```rust
    pub fn banks(&self) -> &HashMap<Slot, BankWithScheduler> {
        &self.banks
    }
```

Reply:
Not quite as simple for `frozen_banks`, but that one may also be better off returning an iterator, since most use-cases seem to just be iterating rather than requireing ownership of the map.

Reply:
> I don't think we need this interface to be entirely unchanging.

good catch: 0aed6268a5e 

> Not quite as simple for frozen_banks, but that one may also be better off returning an iterator, since most use-cases seem to just be iterating rather than requireing ownership of the map.

yeah. you're right. however, I'll pass to cleaning this...